README.md

```diff
@@ -106,14 +115,16 @@ uvicorn app:app --reload
 
 #### Users
 
-| Method  | Endpoint                   | Description        | Authentication Required |
-|---------|----------------------------|--------------------|-------------------------|
-| `POST`  | `/api/v1/users/register`   | Create User        | No                      |
-| `GET`   | `/api/v1/users/{id_}`      | Get User           | Yes                     |
-| `PUT`   | `/api/v1/users/{id_}`      | Update User        | Yes (Owner or Admin)    |
-| `DELETE`| `/api/v1/users/{id_}`      | Delete User        | Yes (Owner or Admin)    |
-| `PATCH` | `/api/v1/users/{id_}`      | Set Role           | Yes (Admin only)        |
-| `GET`   | `/api/v1/users/`           | Get All Users      | Yes (Admin only)        |
+| Method   | Endpoint                       | Description               | Authentication Required |
+|----------|--------------------------------|---------------------------|-------------------------|
+| `POST`   | `/api/v1/users/register`       | Create User               | No                      |
+| `GET`    | `/api/v1/users/{id_}`          | Get User                  | No                      |
+| `PUT`    | `/api/v1/users/{id_}`          | Update User               | Yes (Owner or Admin)    |
+| `DELETE` | `/api/v1/users/{id_}`          | Delete User               | Yes (Owner or Admin)    |
+| `PATCH`  | `/api/v1/users/{id_}`          | Set Role                  | Yes (Admin only)        |
+| `GET`    | `/api/v1/users/`               | Get All Users             | Yes (Admin only)        |
+| `POST`   | `/api/v1/users/reminders`  | Configure daily reminders | Yes (Owner or Admin)    |
```

Comment:
I am not sure that Admin can change user\`s reminder. If you are planning to do so, you\`ll need to add to the endpoint parameter inicating user\`s id you wish to change